src/main/java/odd/jobs/services/UserCrudOperationsService.java

```diff
@@ -26,6 +35,42 @@ public UserDetails loadUserByUsername(String username) throws UsernameNotFoundEx
         return user.orElseThrow(() -> new UsernameNotFoundException("Not found: " + username));
     }
 
+    public List<User> getAllUsers()
+    {
+        List<User> temp=new ArrayList<>();
+        userRepository.findAll().forEach(temp::add);
+        return temp;
+    }
+
+    public User getUserById(Long userId) throws NotFoundException
+    {
+        return userRepository.findById(userId).orElseThrow(() -> new NotFoundException("Nie znaleziono użytkownika"));
+    }
+
+    public User createUser(User user)
+    {
+        return userRepository.save(user);
+    }
+
+    public User updateUser(long id, User user) throws NotFoundException
+    {
+        User updated_user = userRepository.findById(id)
+                .orElseThrow(() -> new NotFoundException("Nie znaleziono użytkownika"));
+        updated_user.setFirstName(user.getFirstName());
+        updated_user.setLastName(user.getLastName());
```

Comment:
obiekty sa niemutowalne nie uzywamy setterow. Przy okazji wywal @Setter z User bo nie wiem skad tam sie to wzielo